src/capi-single-paid/index.js

```diff
@@ -65,4 +67,38 @@ function checkIcon(responseJson) {
     responseJson.videoTag ?
         videoIcon :
         '';
-};
+}
+
+function addSourceset(responseJson) {
+
+  let srcsetFragment = document.createDocumentFragment();
+
+  return responseJson.reduce((sources, source) => {
```

Comment:
Because this has side-effects, it makes more sense to use `forEach` instead of `reduce`. You can then let go of `sources` and just use `srcsetFragment` (just return it after the `forEach`)